src/main/java/org/dwcj/controls/RadioButtonGroup.java

```diff
@@ -6,7 +6,7 @@
 import com.basis.bbj.proxies.sysgui.BBjWindow;
```

Comment:
Remove this at the moment. It is not working anyway and we need to be re-implemented later